.travis.yml

```diff
@@ -17,9 +17,11 @@ matrix:
       otp_release: 20.2
       services: docker
 
-before_install:
-  - sudo apt-get update
-  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+# Install a newer docker version:
+# https://docs.travis-ci.com/user/docker/#Installing-a-newer-Docker-version
+# before_install:
+#   - sudo apt-get update
+#   - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
```

Comment:
i slightly favour of deleting these lines instead of commenting them.

If we need them again in the near future, we remember this PR. If we need these lines in the far future, we need to google for the currently best solution anyways.

Reply:
Cool, sounds good to me 👍 Thanks @tessi!